build/Makefile

```diff
@@ -1,8 +1,9 @@
+include $(BUILD_HELPER_DIR)/tools.mak
 
-DEFAULT_TARGETS := module unit_tests update_generated_files update_system_tests update_examples run_unit_tests wheel sdist
+DEFAULT_TARGETS := module unit_tests update_generated_files update_system_tests update_examples wheel sdist
 export DEFAULT_TARGETS
 
-ADDITIONAL_TARGETS := test flake8
+ADDITIONAL_TARGETS := 
```

Comment:
Can we get rid of this? It doesn't appear to be in use.

Reply:
Removed